cpp/src/arrow/flight/CMakeLists.txt

```diff
@@ -180,10 +176,11 @@ if(ARROW_BUILD_BENCHMARKS)
   set(PERF_PROTO_GENERATED_FILES "${CMAKE_CURRENT_BINARY_DIR}/perf.pb.cc"
                                  "${CMAKE_CURRENT_BINARY_DIR}/perf.pb.h")
 
-  add_custom_command(OUTPUT ${PERF_PROTO_GENERATED_FILES}
-                     COMMAND ${ARROW_PROTOBUF_PROTOC} "-I${CMAKE_CURRENT_SOURCE_DIR}"
-                             "--cpp_out=${CMAKE_CURRENT_BINARY_DIR}" "perf.proto"
-                     DEPENDS ${PROTO_DEPENDS})
+  add_custom_command(
+    OUTPUT ${PERF_PROTO_GENERATED_FILES}
+    COMMAND ${ARROW_PROTOBUF_PROTOC} "-I${CMAKE_CURRENT_SOURCE_DIR}"
+            "--cpp_out=${CMAKE_CURRENT_BINARY_DIR}" "perf.proto"
+    DEPENDS ${PROTO_DEPENDS})
```

Comment:
I don't see the intent of the changes in this file.

Reply:
no intent, accidental commit from playing w/ formatting. Reverting